### PR TITLE
Implement a basic `Bloody Caress` mob TP skill

### PR DESCRIPTION
**Pull request by mrhappyasthma**
There is not a whole lot of info about this spell available online, so I implemented a basic version similar to `Goblin Rush`, which is also a physical 3-hit spell.

I confirmed the animation matches retail videos from youtube.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/mobskills/bloody_caress.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 3
+    local accmod = 1
+    local dmgmod = 1
+
+    -- TODO: Once `Floral Bouquet` TP move is implemented, this skill is eligible to target
```
Comment (TeoTwawki) [inline scripts/globals/mobskills/bloody_caress.lua:26]: Ty for the todo note, and thank you for conservative mod data on initial implementation.

**Comment by mrhappyasthma** [comment]
Done as part of https://github.com/LandSandBoat/server/issues/202

**Review by zach2good — CHANGES_REQUESTED** [review]
It would be good if you also left something like this comment in the header for future readers:
```
There is not a whole lot of info about this spell available online, so I implemented a basic version similar to Goblin Rush, which is also a physical 3-hit spell.
```

**Comment by mrhappyasthma** [comment]
Added the note. Please take another look.

**Review by zach2good — APPROVED** [review]

**Pull request merged**